Functions/src/main/java/com/epam/engx/cleancode/functions/task1/RegisterAccountAction.java

```diff
@@ -13,26 +13,28 @@
 
     private PasswordChecker passwordChecker;
     private AccountManager accountManager;
+    private static final int LENGTH_NAME =  5;
+    private static final int PASSWORD_LENGTH = 8;
 
     public void register(Account account) {
-        if (account.getName().length() <= 5){
+        verificationPasswordAndName(account);
+        account.setCreatedDate(new Date());
+        List<Address> addresses = Arrays.asList(account.getHomeAddress(), account.getWorkAddress(), account.getAdditionalAddress());
+        account.setAddresses(addresses);
+        accountManager.createNewAccount(account);
+    }
+
+    private void verificationPasswordAndName(Account account) {
+        if (account.getName().length() <= LENGTH_NAME) {
```

Comment:
Please analyze the register(Account account) method with the Single Responsibility Principle.